.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,246 @@
+name: Run automated system tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+      - support/**
+    types: [opened, synchronize]
+    paths-ignore:
+      - '**/*.md'
+  release:
+    types:
+      - published
+  workflow_dispatch:
+    inputs:
+      neofs_testcases_ref:
+        description: 'neofs-testcases ref. Default ref - latest master. Examples: v0.36.0, 8fdcc6d7e798e6511be8806b81894622e72d7fdc, branch_name'
+        required: false
+        default: ''
+
+permissions: write-all
+
+jobs:
+  run_system_tests:
+    runs-on: ubuntu-latest
+    timeout-minutes: 500
+    steps:
+      - name: Get the current date
+        id: date
+        run: echo "::set-output name=timestamp::$(date +%s)"
+
+      - name: Set RUN_ID
+        env:
+          TIMESTAMP: ${{ steps.date.outputs.timestamp }}
+        run: echo "RUN_ID=${{ github.run_number }}-$TIMESTAMP" >> $GITHUB_ENV
+
+      - name: Checkout neofs-testcases repository
+        uses: actions/checkout@v3
+        with:
+          path: neofs-testcases
+
+      - name: Checkout neofs-node repository
+        uses: actions/checkout@v3
+        with:
+          repository: nspcc-dev/neofs-node
+          ref: 'master'
```

Comment:
Mb latest release?

Reply:
It is a good question. On the one side - 'master' will allow us to always keep tests in 'current' state, since we won't be able to merge if smth is broken. On the other side - fixed release version will allow us to prevent unnecessary building here + tests will be more stable and consistent, since we run them on a well-known version.
@roman-khimov what do you think?

Reply:
I added fixed release versions for every binary except s3 gw, current available release is too old, so I used fixed commit.